src/main/java/com/diffblue/cover/annotations/generated/MethodUnderTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 Diffblue Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.diffblue.cover.annotations.generated;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Indicates the method being tested in the annotated test method. Diffblue Cover attaches this
+ * annotation to all tests that it creates.
+ */
+@Retention(RUNTIME)
```

Comment:
:bulb: Techincally, we need this only at source level - we're only using this as some kind of meta-data for the combiner and related code.

Reply:
Won't we be reading it from bytecode classes, therefore needing at least class retention?

Reply:
Nope, this is purely on the source code level. Remember this goes only into test classes, and we obviously have those files (since we write them!). No need to look at bytecode here.

Reply:
Ok, updated